eng/Versions.props

```diff
@@ -12,11 +12,11 @@
     <SystemRuntimeWindowsRuntimePackageVersion>4.6.0</SystemRuntimeWindowsRuntimePackageVersion>
     <SystemManagementPackageVersion>5.0.0</SystemManagementPackageVersion>
     <SystemThreadingTasksExtensionsPackageVersion>4.5.4</SystemThreadingTasksExtensionsPackageVersion>
-    <SystemMemoryPackageVersion>4.5.4</SystemMemoryPackageVersion>
+    <SystemMemoryPackageVersion>4.5.5</SystemMemoryPackageVersion>
```

Comment:
Is there a reason why we have to make these changes together with the nca3.1 changes? I think it would be better to isolate the two.

Reply:
It's actually the other way round: The package updates require the nca3.1 changes, because the newer packages are not supported on nca3.1 any more. 

It's probably possible to separate the two (and merge the 3.1 change first), if that's really required.